README.md

```diff
@@ -74,6 +75,14 @@ python setup.py install
       ...
     ```
 
+* `chunk-size`
+
+    By default table migrations are broken into chunks, a number of rows at a
+    time. The default size is `10000`. This option with change the chunk size.
+
+    Setting `chunk-size` to `0` will disable chunking and the migration will
+    migrate the while table in one go. Be warned that this will use more RAM.
```

Comment:
This may also cause large tables to not migrate due to MariaDB transaction limits, but I'm not certain if we want to turn the README into a dirge.

Reply:
I think it would be good to mention, after all that's the main reason for the chunked migration